index.html

```diff
@@ -354,13 +354,17 @@ <h2 class="about-me-title">About me</h2>
           to chat please get in touch.
         </p>
 
-        <form action="https://formspree.io/f/xlezlyqd" method="POST">
+        <form
+          id="contact-form"
+          action="https://formspree.io/f/xlezlyqd"
+          method="POST"
+        >
```

Comment:
- [ ] As you can see on the screenshot made, the form does not work as expected

- Even When any input field are not fill in yet I can still get this error message instead to the alert message from `required` attribute from input 👇🏻 

![zzz22222](https://user-images.githubusercontent.com/79058364/162034441-015b2474-0f81-432a-8369-57a11b6d94c4.png)


- And even after all fields are filling correctly, I'm not able to submit the form because I'm still getting the error message👇🏻 Kindly fix this 


![zzzzzzzz3333333333](https://user-images.githubusercontent.com/79058364/162036651-ba692e6d-9e85-466a-ada1-e962232a74db.png)